File: Back/Apis/Api_Py/src/settings/dominio/variables_dominio.py
#Librerias Instaladas
from dotenv import load_dotenv; 

#Modulos Nativos
import os;

load_dotenv(); 

DOMINIO = os.environ.get("DOMINIO"); 
DOMINIO_ATRIBUTOS = os.environ.get("DOMINIO_ATRIBUTOS");
DOMINIO_URL = os.environ.get("DOMINIO_URL");
DOMINIO_BASE_DN = os.environ.get("DOMINIO_BASE_DN");
DOMINIO_USUARIO_PRUEBA = os.environ.get("DOMINIO_USUARIO_PRUEBA");
DOMINIO_USUARIO_PRUEBA_CONTRASENA = os.environ.get("DOMINIO_CONTRASENA_USUARIO_PRUEBA");
DOMINIO_PUERTO = os.environ.get("DOMINIO_PUERTO");
DOMINIO_FILTRO_BUSQUEDA = os.environ.get("DOMINIO_FILTRO_BUSQUEDA");
DOMINIO_GRUPOS_EXCLUIDOS = os.environ.get("DOMINIO_GRUPOS_EXCLUIDOS"); 

File: Back/Apis/Api_Js/src/settings/dominio/variablesDominio.js
//Librerias
import { config } from "dotenv";

config();

export const dominio = process.env.DOMINIO; 
export const dominio_url = process.env.DOMINIO_URL; 
export const dominio_base_dn = process.env.DOMINIO_BASE_DN; 
export const dominio_usuario_prueba = process.env.DOMINIO_USUARIO_PRUEBA; 
export const dominio_contrasena_usuario_prueba = process.env.DOMINIO_CONTRASENA_USUARIO_PRUEBA; 
export const dominio_filtro_busqueda = process.env.DOMINIO_FILTRO_BUSQUEDA; 
export const dominio_atributos = process.env.DOMINIO_ATRIBUTOS; 
export const dominio_puerto = process.env.DOMINIO_PUERTO; 
export const dominio_grupos_excluidos = process.env.DOMINIO_GRUPOS_EXCLUIDOS; 